src/components/searchUserAside/index.js

```diff
@@ -8,14 +8,18 @@ import UsersList from "../usersList"
 import UserSearchBar from "../userSearchBar/UserSearchBar"
 
 import { getUserByName } from "../../service/apiClient"
+import { useTranslation } from "react-i18next"
 
 import "./style.css"
+import { t } from "i18next"
```

Comment:
Don't think you need this as you have defined it already down below.

Reply:
Oh yeah! I should have use useTranslation instead 🤦‍♂️